lib/restbase.js

```diff
@@ -243,6 +243,10 @@ RESTBase.prototype._request = function(req) {
     var match = priv.router.route(childReq.uri);
     var methods = match && match.value && match.value.methods;
     var handler = methods && (methods[childReq.method] || methods.all);
+    if (childReq.method === 'head' && !handler) {
```

Comment:
Putting a Boolean will work just as well. If we change `childReq`'s method, we might as well set a simple Boolean and check it on each request. If the method hasn't changed, no Boolean property is set, which will evaluate to `false` (well, `not true` to be precise).